settings.gradle

```diff
@@ -6,7 +6,8 @@ include 'h2o-web'
 include 'h2o-app'
 include 'h2o-r'
 include 'h2o-py'
-include 'h2o-assembly'
+include 'h2o-assemblies:main'
```

Comment:
as far as I can see, it does. The gradle build completes successfully; builds both versions of h2o.jar; and both jars can start without throwing any exceptions